tests/table/playerCards.test.ts

```diff
@@ -153,8 +154,10 @@ describe("Player cards", function () {
                 Money: 10000,
             }];
             const actions: GameActionStartInformation[] = [];
+            authenticated(false);
```

Comment:
Why this change?

Reply:
Because subscribe notifies only if value is not the same as previous, in some testst authenticated already was true, so we should "logout" first and then "login"

Reply:
How this value could be initially true if we never set it to true previously

Reply:
i don't know, but without this some tests doesn't work

Reply:
Your tests are interfere with each other, since we use global `authManager`. So value of `authenticated` kept true from previous tests. In order to make controlled tests use `beforeAll`, set `authenticated(false)` in that method, so each test would start with not authenticated user.